yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/server/CodeVersion.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * CodeVersion.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2025 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.relational.yamltests.server;
+
+/**
+ * Marker interface representing a code version.
+ */
+public interface CodeVersion extends Comparable<CodeVersion> {
```

Comment:
Is this used anymore? I think it can be deleted.

Reply:
Whoops, meant to remove it earlier. Done